Jenkinsfile.examples

```diff
@@ -0,0 +1,36 @@
+node {
+    def SOURCEDIR = pwd()
+    def TEST_CONTAINER = 'examples/java-test'
+    def DOCKERFILE_DIR = './examples/docker'
+    def PUBLISHED_VERSION = '1.7.3'
```

Comment:
If there's release instructions somewhere, can we get them updated to ensure that this gets bumped too?

Reply:
Good idea.  I think that it can be added to the binding publish script, so that it's automatic.